pset-functions.js

```diff
@@ -175,6 +318,51 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+const wordToTest = "racecar"
+
+isPalindrome(wordToTest)
+
+function isPalindrome(word) {
+
+  const arrayForward = word.split('')
+  const arrayReverse = word.split('').reverse()
+
+    if (arrayForward.toString() === arrayReverse.toString()) {
+
+      console.log(`${word} isPalindrome = true`)
+    }
+    else {
+
+    console.log(`${word} isPalindrome = false`)
+
+    }
+}
```

Comment:
@thetripler Good work here coming up with this solution. I see you're splitting and reversing and comparing the reversed string with the current string. One suggestion since you want to compare the `word` with the reverse of it you don't have to split the word.` arrayReverse` there is a method that would allow you to turn an array into a string and that's called [join](https://www.w3schools.com/jsref/jsref_join.asp). Also for returning true or false you can write the following `return false` or `return true` the console.log shows the message but you would want to write your return of true and false like that.